tests/utils.go

```diff
@@ -2499,7 +2508,33 @@ func GetPolicyMatchedToVmi(name string, vmi *v1.VirtualMachineInstance, namespac
 	}
 
 	applyLabels(policy.Spec.Selectors.VirtualMachineInstanceSelector, vmi.Labels, matchingVmiLabels)
-	applyLabels(policy.Spec.Selectors.NamespaceSelector, namespace.Labels, matchingNSLabels)
+	applyLabels(policy.Spec.Selectors.NamespaceSelector, namespaceLabels, matchingNSLabels)
+
+	if namespace != nil {
+		namespace.Labels = namespaceLabels
+	}
+
+	return policy
+}
+
+// PreparePolicyAndVMI mutates the given vmi parameter by adding labels to it. Therefore, it's recommended
+// to use this function before creating the vmi. Otherwise, its labels need to be updated.
+func PreparePolicyAndVMI(vmi *v1.VirtualMachineInstance) *migrationsv1.MigrationPolicy {
+	return PreparePolicyAndVMIWithNsAndVmiLabels(vmi, nil, 1, 0)
+}
+
+func PreparePolicyAndVMIWithBandwidthLimitation(vmi *v1.VirtualMachineInstance, bandwidth resource.Quantity) *migrationsv1.MigrationPolicy {
+	policy := PreparePolicyAndVMI(vmi)
+	policy.Spec.BandwidthPerMigration = &bandwidth
+
+	return policy
+}
+
+func CreateMigrationPolicy(virtClient kubecli.KubevirtClient, policy *migrationsv1.MigrationPolicy) *migrationsv1.MigrationPolicy {
```

Comment:
That's 4 Policy functions in a row in this overloaded file, can they maybe move to their own policy.go somewhere? That way we'd reduce the size of this file instead of increasing it...

Reply:
done